{{cookiecutter.project_shortname}}/.travis.yml

```diff
@@ -53,7 +53,7 @@ before_install:
 
 install:
   - "travis_retry pip install -r .travis-${REQUIREMENTS}-requirements.txt"
-  - "travis_retry pip install -e .[all,postgresql,elasticsearch2]"
+  - "travis_retry pip install -e .[all,postgresql,{{cookiecutter.elasticsearch}}]"
```

Comment:
Maybe for this it makes sense to use the same version as ES_VERSION, just need to find the way to define variables in cookiecutter without getting them from the input.

Reply:
You can define [dictionaries](https://cookiecutter.readthedocs.io/en/latest/advanced/dict_variables.html) in the cookiecutter.json which you can then use in templates.

Reply:
Thanks @slint. I used an if else statement because when I passed a dictionary, the message shown in the genaration wasn't showing the options:
```
...
elasticsearch [default]:
```